app/utils/security.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from 'mattermost-redux/client';
+import CookieManager from 'react-native-cookies';
+import urlParse from 'url-parse';
+
+export function setCSRFFromCookie(url) {
```

Comment:
Is there any reason the calling code might want to wait for this to happen instead of having this done asynchronously?

Reply:
If the CSRF is not properly set in the Client4 and any non-GET requests will be made, they will return a 401.

Reply:
401s actually cause the app to log out since it thinks the session has expired, so we should probably wait for this to be called

https://github.com/mattermost/mattermost-redux/blob/master/src/actions/helpers.js#L18